dev Docker Image missing redshift driver

**Describe the bug**
When trying to run the latest build, I am unable to connect to redshift, with a missing dependency error.

```
cube-refresh_1        | Cube.js Error ---------------------------------------
cube-refresh_1        | 
cube-refresh_1        | Error: Cannot find module '@cubejs-backend/redshift-driver'
```

Looks like https://github.com/cube-js/cube.js/blob/d40d13b0a80cd65b27337e87a523314af585dbc6/packages/cubejs-docker/dev.Dockerfile 

Has omitted the redshift driver package.

**To Reproduce**
Run the dev image while using the redshift driver.  In my case, I am actively importing it inside the config.

**Expected behavior**
Should be able to connect to redshift.

--- a/packages/cubejs-docker/dev-alpine.Dockerfile
+++ b/packages/cubejs-docker/dev-alpine.Dockerfile
@@ -50,6 +50,7 @@ COPY packages/cubejs-mssql-driver/package.json packages/cubejs-mssql-driver/pack
 COPY packages/cubejs-mysql-driver/package.json packages/cubejs-mysql-driver/package.json
 COPY packages/cubejs-cubestore-driver/package.json packages/cubejs-cubestore-driver/package.json
 COPY packages/cubejs-oracle-driver/package.json packages/cubejs-oracle-driver/package.json
+COPY packages/cubejs-redshift-driver/package.json packages/cubejs-redshift-driver/package.json
 COPY packages/cubejs-postgres-driver/package.json packages/cubejs-postgres-driver/package.json
 COPY packages/cubejs-questdb-driver/package.json packages/cubejs-questdb-driver/package.json
 COPY packages/cubejs-materialize-driver/package.json packages/cubejs-materialize-driver/package.json
@@ -103,6 +104,7 @@ COPY packages/cubejs-mssql-driver/ packages/cubejs-mssql-driver/
 COPY packages/cubejs-mysql-driver/ packages/cubejs-mysql-driver/
 COPY packages/cubejs-cubestore-driver/ packages/cubejs-cubestore-driver/
 COPY packages/cubejs-oracle-driver/ packages/cubejs-oracle-driver/
+COPY packages/cubejs-redshift-driver/ packages/cubejs-redshift-driver/
 COPY packages/cubejs-postgres-driver/ packages/cubejs-postgres-driver/
 COPY packages/cubejs-questdb-driver/ packages/cubejs-questdb-driver/
 COPY packages/cubejs-materialize-driver/ packages/cubejs-materialize-driver/

--- a/packages/cubejs-docker/dev.Dockerfile
+++ b/packages/cubejs-docker/dev.Dockerfile
@@ -53,6 +53,7 @@ COPY packages/cubejs-mssql-driver/package.json packages/cubejs-mssql-driver/pack
 COPY packages/cubejs-mysql-driver/package.json packages/cubejs-mysql-driver/package.json
 COPY packages/cubejs-cubestore-driver/package.json packages/cubejs-cubestore-driver/package.json
 COPY packages/cubejs-oracle-driver/package.json packages/cubejs-oracle-driver/package.json
+COPY packages/cubejs-redshift-driver/package.json packages/cubejs-redshift-driver/package.json
 COPY packages/cubejs-postgres-driver/package.json packages/cubejs-postgres-driver/package.json
 COPY packages/cubejs-questdb-driver/package.json packages/cubejs-questdb-driver/package.json
 COPY packages/cubejs-materialize-driver/package.json packages/cubejs-materialize-driver/package.json
@@ -112,6 +113,7 @@ COPY packages/cubejs-mssql-driver/ packages/cubejs-mssql-driver/
 COPY packages/cubejs-mysql-driver/ packages/cubejs-mysql-driver/
 COPY packages/cubejs-cubestore-driver/ packages/cubejs-cubestore-driver/
 COPY packages/cubejs-oracle-driver/ packages/cubejs-oracle-driver/
+COPY packages/cubejs-redshift-driver/ packages/cubejs-redshift-driver/
 COPY packages/cubejs-postgres-driver/ packages/cubejs-postgres-driver/
 COPY packages/cubejs-questdb-driver/ packages/cubejs-questdb-driver/
 COPY packages/cubejs-materialize-driver/ packages/cubejs-materialize-driver/

